lib/solaar/cli/unpair.py

```diff
@@ -28,13 +28,12 @@ def run(receivers, args, find_receiver, find_device):
 	dev = find_device(receivers, device_name)
 
 	if not dev.receiver.may_unpair:
-		print('Receiver for %s [%s:%s] does not unpair' % (dev.name,dev.wpid,dev.serial))
-		return
+		print('Receiver for %s [%s:%s] does not unpair, but attempting anyway' % (dev.name,dev.wpid,dev.serial))
 
-	# query these now, it's last chance to get them
 	try:
+		# query these now, it's last chance to get them
 		number, codename, wpid, serial  = dev.number, dev.codename, dev.wpid, dev.serial
-		del dev.receiver[number]
+		dev.receiver._delete(number,True) # force an unpair
```

Comment:
Coding style: space between `number,` and `True`?
And maybe name it `_unpair_device` or something more clear than `_delete`?

Reply:
OK, done.